src/CMakeLists.txt

```diff
@@ -199,6 +201,34 @@ INSTALL (FILES ${PROJECT_BINARY_DIR}/${BINARY_NAME}.timer
             DESTINATION usr/lib/systemd/system)
 
 
+###### I18N FILES ######
+
+SET(_locales
+    zh_CN
+    )
+
+IF(_locales)
```

Comment:
no need for this condition, this is always true unless someone removes locales at all.